.github/workflows/CI.yml

```diff
@@ -3,22 +3,29 @@ name: CI
 on:
   push:
     paths-ignore:
-      - '.travis.yml'
-      - 'filecopy.bat'
+      - '.github/*'
+      - '.github/*_TEMPLATE/**'
+      - '*.bat'
+      - '*.yml'
```

Comment:
Doesn't that block github's own CI .yml changes?

Reply:
It only does it for the root directory. It would need to be `'**/*.yml'` to be recursive.